python/cusignal/filtering/filtering.py

```diff
@@ -695,6 +695,10 @@ def channelize_poly(x, h, n_chans):
 
     # number of taps in each h_n filter
     n_taps = int(len(h) / n_chans)
+    if n_taps > 32:
+        raise NotImplementedError('The number of taps in each filter is \
```

Comment:
Can we change this to include current number of taps like the previous implementation?
Because `n_taps` is calculated, users might know what it is at the time of the error.

Reply:
Good idea. I also changed the wording to "the number of calculated filter taps"